Gemfile.lock

```diff
@@ -26,9 +26,9 @@ GEM
       slop (~> 3.4)
     rainbow (2.0.0)
     rake (10.4.2)
-    rubocop (0.31.0)
+    rubocop (0.33.0)
```

Comment:
Yes, update was required because the `DisabledByDefault` option was added in version 0.33 - https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#new-features-1